README.md

```diff
@@ -1,23 +1,145 @@
+  </a>
+</p>
 
-## Init Husky
+<br/>
+
+## 📚 About
+
+<strong> Landing Page</strong> is a page that has all the elements aimed at converting the visitor into Lead, opportunity or customer. Also as landing pages or conversion pages, Landing Pages are widely used in Digital Marketing campaigns, as they tend to have high conversion rates.
+
+## 🛠 Design pattern
+
+<p align="justify">
+<a href="https://github.com/ttoss/monorepo"><strong> Monorepo </strong></a> (ttoss Monorepo Template) as the name says, would be a single repository to keep the code of several projects. Each app domain is within this repository: backend, IoT, machine learning, web and mobile apps. This repository structure is useful because we can access any part of the application without changing the repository.
+</p>
+
+<br/>
+
+## 🚀 Projects
+
+- **[Simulare](https://github.com/TriangulosTecnologia/lps/tree/main/lps/simulare)**
+
+<br/>
+
+## 📌 Technologies
+
+- [TypeScript](https://www.typescriptlang.org/)
+- [Next.js](https://nextjs.org/)
+- [Fullpage.js](https://alvarotrigo.com/react-fullpage/)
+- [ThemeUI](https://theme-ui.com/)
+- [React Hook Form](https://react-hook-form.com/)
+- [Yup](https://github.com/jquense/yup)
+- [React.js](https://reactjs.org/)
+- [ESlint](https://eslint.org/)
+- [Husky](https://typicode.github.io/husky/#/)
+- [Commitlint](https://commitlint.js.org/#/)
+- [Jest](https://jestjs.io/)
+- [Lint Staged](https://github.com/okonet/lint-staged#readme)
+- [NPM Check Updates](https://github.com/raineorshine/npm-check-updates)
+- [Prettier](https://prettier.io/)
+- [Stylelint](https://stylelint.io/)
+
+<br/>
+
+## 🔖 Design
+
+We've adopted some specifications to define our <a href="https://theme-ui.com/theming/"><strong> Theme UI theme </strong></a>. You can check our <a href="https://github.com/TriangulosTecnologia/lps/blob/main/lps/simulare/src/theme.ts"><strong> full theme specifications here</strong></a>.
+
+</p>
+
+- ## Breakpoints
+
+We defined six breakpoints `['40rem', '48rem', '64rem', '80rem', '96rem', '120rem']`, which is equivalent to screens with `[640, 768, 1024, 1280, 1536, 1920]` pixels, if the root font size is equal to `16px`. Defining this array of breakpoints comes in handy because we can work with <a href="https://styled-system.com/guides/array-props/"><strong>array props </strong></a> when we need to implement responsiveness.
+
+- ## Font Sizes
+
+We defined our font size based on <a href="https://styled-system.com/guides/array-props/"><strong>Tailwind CSS </strong></a> convention.
+
+```js
+
+fontSizes: {
+  xs: '0.75rem',
+  sm: '0.875rem',
+  base: '1rem',
+  lg: '1.125rem',
+  xl: '1.25rem',
+  '2xl': '1.5rem',
+  '3xl': '1.875rem',
+  '4xl': '2.25rem',
+  '5xl': '3rem',
+  '6xl': '3.75rem',
+  '7xl': '4.5rem',
+  '8xl': '6rem',
+  '9xl': '8rem',
+}
 
 ```
-npx husky-init && yarn
+
+If we want that some text has `sm` for the first screen (< 40rem), and `lg` for the subsequent screen sizes.
```

Comment:
If we want some text has sm for the first screen (< 40rem) and lg for the following screen sizes, then we can define a text component with the following style: